docs/development/releasing.rst

```diff
@@ -12,231 +12,249 @@ more of the process, if not all.
+#. Edit the ``CHANGES.rst`` file by changing the date for the version you are
+   about to release from "unreleased" to today's date.  Also be sure to remove
+   any sections of the changelog for that version that have no entries.
 
- 7. Now install Astropy into the virtualenv::
+#. Add the changes to ``CHANGES.rst`` and ``setup.py``::
 
-        $ python setup.py install
+      $ git add CHANGES.rst setup.py
 
-    This is necessary for two reasons.  First, the entry points for the
-    releaser scripts need to be available, and these are in the Astropy
-    package. Second, the build process will generate .c files from the
-    Cython .pyx files, and the .c files are necessary for the source
-    distribution.
+   and commit with message::
 
- 8. Install zest.releaser into the virtualenv; use ``--upgrade --force`` to
-    ensure that the latest version is installed in the virtualenv (if you're
-    running a csh variant make sure to run ``rehash`` afterwards too)::
+      $ git commit -m "Preparing release v<version>"
 
-        $ pip install zest.releaser==3.49 --upgrade --force
+#. Tag the commit with ``v<version>``, being certain to sign the tag with the
+   ``-s`` option::
 
-    .. note::
+      $ git tag -s v<version>
 
-        zest.releaser > 3.49 has a still open issue that prevents our release
-        code from correctly updating the ``VERSION`` variable in our ``setup.py``;
-        see `zestsoftware/zest.releaser#62 <https://github.com/zestsoftware/zest.releaser/pull/62>`_.
+#. Edit the ``VERSION`` in ``setup.py`` to be the next version number, but with
+   a ``.dev`` suffix at the end (E.g., ``0.2.3.dev``).
 
- 9. Ensure that all changes to the code have been committed, then start the
-    release by running::
+#. Also update the ``CHANGES.rst`` file with a new section for the next version.
+   You will likely want to use the ``add_to_changelog.py`` script in the
+   `astropy-tools repository`_ for this.
 
-        $ fullrelease
+#. Add the changes to ``CHANGES.rst`` and ``setup.py``::
 
- 10. You will be asked to enter the version to be released.  Press enter to
-     accept the default (which will normally be correct) or enter a specific
-     version string.  A diff will then be shown of CHANGES.rst and setup.py
-     showing that a release date has been added to the changelog, and that the
-     version has been updated in setup.py.  Enter 'Y' when asked to commit
-     these changes.
+      $ git add CHANGES.rst setup.py
 
- 11. You will then be shown the command that will be run to tag the release.
-     Enter 'Y' to confirm and run the command.
+   and commit with message::
 
- 12. When asked "Check out the tag (for tweaks or pypi/distutils server
-     upload)" enter 'N': zest.releaser does not offer enough control yet over
-     how the register and upload are performed so we will do this manually
-     until the release scripts have been improved.
+      $ git commit -m "Back to development: v<next_version>"
```

Comment:
Maybe clarify that ``<next_version>`` should be followed by ``.dev`` in the commit message, if necessary? (``Back to development: v1.3.0.dev``)